it/uk/gov/hmrc/personaldetailsvalidation/pages/PersonalDetailsPage.scala

```diff
@@ -64,4 +73,11 @@ case class PersonalDetailsPage(completionUrl: String, verifyThisPageDisplayedWit
       case Some(element) => click on element
       case _ => fail("Continue button not found")
     }
+
+  def selectPostcodeOption(): Unit =
+    find(linkText("I don't have a National Insurance number")) match {
```

Comment:
Can we use id or better css selector? You don't want this logic to break just because of the text change.